site/content/entry/citra-progress-report-2017-august.md

```diff
@@ -0,0 +1,159 @@
++++
+date = "2017-09-04T15:07:00-04:00"
+title = "Citra Progress Report - 2017 August"
+tags = [ "progress-report" ]
+author = "anodium"
+coauthor = "saphiresurf"
+forum = 0000
++++
+
+2017 has been an amazing year, with more work having been put into the project
+than ever before, but it's not over yet! Last we met was
+[June](/entry/citra-progress-report-2017-june/), and just two months later the
+[Citra issue tracker](https://github.com/citra-emu/citra/pulls) is brimming with
+lots of changes once more. I am extremely excited for this month (and what might
+be of the next month or two) but we're getting ahead of ourselves! On this
```

Comment:
Awkward wording for "and what might be of the next month or two"... took me a few takes to understand what you're trying to say. How about just: "and what's coming up in the next few months"?